README.md

```diff
@@ -482,19 +569,19 @@ Before following these instructions, ensure the existing model you are adding sk
    ...
    ```
 
-   The synthetic data set will be three files in the newly created `generated` directory named `generated*.json`, `test*.jsonl`, and `train*.jsonl`.
-
-> [!NOTE]
-> If you want to pickup from where a failed or canceled `ilab data generate` left off, you can copy the
-> `generated*.json` file into a file named `regen.json`. `regen.json` will be picked up at the start of `lab
-> generate` when available. You should remove it when the process is completed.
+   The synthetic data set will be three files in the newly created `~/.local/share/instructlab/datasets` directory named `skills_train_msgs_*.jsonl` and `knowledge_train_msgs_*.jsonl`
 
-2. Verify the files have been created by running the `ls generated` command.
+2. Verify the files have been created by running the `ls datasets` command. Note: you must be in your XDG_DIR/instructlab directory.
```

Comment:
May not be XDG_DIR on MacOS - in this case it would be `Library/Application\ Support/instructlab`

Reply:
XDG_DATA_HOME? (need to check if the right variable - but definitely there's no envvar called XDG_DIR).